packages/react-native-reanimated/Common/cpp/reanimated/LayoutAnimations/LayoutAnimationsManager.cpp

```diff
@@ -24,6 +24,11 @@ void LayoutAnimationsManager::configureAnimationBatch(
         enteringAnimationsForNativeID_[tag] = config;
         continue;
       }
+      if (type == LAYOUT_AND_STYLE) {
```

Comment:
Why you can't reuse the logic from `getConfigsForType`?

Reply:
`getConfigsForType` of LAYOUT_AND_STYLE (renamed STYLE_TRANSITION) returns config for LAYOUT animation:
```cpp
    case LAYOUT:
    case STYLE_TRANSITION: // a subset of LAYOUT
      return layoutAnimations_;
    case SHARED_ELEMENT_TRANSITION:
    case SHARED_ELEMENT_TRANSITION_PROGRESS:
      return sharedTransitionAnimations_;
```
I want to leverage code of LAYOUT as much as possible, and STYLE_TRANSITIONS  are a subset of LAYOUT animations. Therefore I use separate array to distinguish the LAYOUT animations being STYLE_TRANSITIONS. The similar approach was used for SHARED_ELEMENT_TRANSITION_PROGRESS.